io.github.Pithos.json

```diff
@@ -34,15 +41,34 @@
         "sha256": "e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020"
       }]
     },
+    {
+      "name": "libnotify",
+      "buildsystem": "meson",
+      "cleanup": ["/bin"],
+      "config-opts": [
+        "-Dtests=false",
+        "-Dintrospection=disabled",
+        "-Dman=false",
+        "-Dgtk_doc=false",
+        "-Ddocbook_docs=disabled"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.9.tar.xz",
+          "sha256": "66c0517ed16df7af258e83208faaf5069727dfd66995c4bbc51c16954d674761"
+        }
+      ]
+    },
     {
       "name": "pithos",
       "buildsystem": "meson",
       "cleanup": ["/share/man"],
       "sources": [
         {
           "type": "git",
-          "url": "https://github.com/pithos/pithos.git",
-          "commit": "1.5.1"
+          "url": "https://github.com/rcalixte/pithos.git",
```

Comment:
What?

Reply:
testing the build... this was a temporary commit